src/errorHandler.js

```diff
@@ -0,0 +1,39 @@
+const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const VALIDATION_ERRORS = {
+  NO_TEXT:
+    'Text to convert is required. Correct request is: ' +
+    '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  NO_TARGET_CASE:
+    '"toCase" query param is required. Correct request is: ' +
+    '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  CASE_NOT_SUPPORTED:
+    'This case is not supported. Available cases: ' +
+    `${availableCases.join(', ')}.`,
+};
+
+function errorHandler(originalText, targetCase) {
+  const errors = [];
+
+  if (!originalText) {
+    errors.push({
+      message: VALIDATION_ERRORS.NO_TEXT,
+    });
+  }
+
+  if (!availableCases.includes(targetCase) && targetCase) {
```

Comment:
The condition `!availableCases.includes(targetCase) && targetCase` is slightly misleading. It checks if the `targetCase` is not supported and if it exists. However, this logic can be simplified by checking for `targetCase` first, then verifying if it's included in `availableCases`. Consider reordering the conditions for clarity.